homeassistant/components/shelly/switch.py

```diff
@@ -95,6 +95,8 @@ async def async_setup_rpc_entry(
 class BlockRelaySwitch(ShellyBlockEntity, SwitchEntity):
     """Entity that controls a relay on Block based Shelly devices."""
 
+    _attr_device_class = SwitchDeviceClass.SWITCH
```

Comment:
This is not needed, as that is considered the default.

Reply:
Without device_class the switch currently uses mdiFlash though, which is rather confusing.

But, given the fact Paulus also commented on the Pr I suggested to change that also, I am confused what to do...